internal/vrfroute/vrfroute.go

```diff
@@ -0,0 +1,52 @@
+package vrfroute
+
+import (
+	metal "github.com/equinix/equinix-sdk-go/services/metalv1"
+	"github.com/equinix/metal-cli/internal/outputs"
+	"github.com/spf13/cobra"
+)
+
+type Client struct {
+	Servicer Servicer
+	Service  metal.VRFsApiService
+	Out      outputs.Outputer
+}
+
+func (c *Client) NewCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   `vrf route`,
```

Comment:
If the route operations will be nested under `vrf`, this command shouldn't exist.  Since we decided to nest BGP dynamic neighbor operations under the `gateways` sub commands, I think it makes sense to align with that here and nest the route operations under the `vrf` subcommand.